### ISSUE TITLE
no custom html output when stderr used

### ISSUE THREAD

**Issue by dheise-rheinenergie-trading-com** [jhuckaby/Cronicle#516]
## Summary

My shell script writes custom html content to standard out. Normally this output is rendered in the log. If my script also writes output to standard error, then my custom html is not shown anymore.

## Steps to reproduce the problem
new event with shell plugin

script:
```
#!/bin/bash

echo "{\"html\":{\"title\":\"my title\",\"content\":\"<a href='http:/google.de' target='_blank'>link</a><br/>\"}}" 
echo This message goes to stdout
```
custom html content is shown
![image](https://user-images.githubusercontent.com/73586581/185336168-409142dc-5461-45e3-a8e7-97478834e655.png)

```
#!/bin/bash

echo "{\"html\":{\"title\":\"my title\",\"content\":\"<a href='http:/google.de' target='_blank'>link</a><br/>\"}}" 
echo This message goes to stdout
echo This message goes to stderr >&2
```
custom html content is not shown. I only see the error output.
![image](https://user-images.githubusercontent.com/73586581/185336247-0763903e-6cee-4432-99e9-bfcb503b16d4.png)

### Operating system and version?
RedHat 7

### Node.js version?
v14.20

### Cronicle software version?
Fork cronicle-edge 1.5.10.2

**Comment by jhuckaby** [comment]
This is not Cronicle.  You are using a 3rd party fork.  I cannot offer support for it. 
 Please submit your bug over at https://github.com/cronicle-edge/cronicle-edge/issues

**Comment by mikeTWC1984** [comment]
It's actually inherited from original cronicle. That "error output" is html itself and overrides user defined html:
https://github.com/jhuckaby/Cronicle/blob/master/bin/shell-plugin.js#L87:L91

**Comment by dheise-rheinenergie-trading-com** [comment]
Hi @jhuckaby,
Do you think you can take a look at the code Mike mentioned?

Daniel

**Comment by jhuckaby** [comment]
Oh yeah, sorry guys.  I didn't recognize those screenshots so I wasn't able to put it together.  I see now -- thanks for the code link, @mikeTWC1984.

I guess I can make it conditionally do the STDERR error report, i.e. only if your code hasn't already passed in an `html` property.

This will be fixed in the next release.  Re-opening issue.

Thanks!

**Comment by jhuckaby** [comment]
Fixed in [v0.9.10](https://github.com/jhuckaby/Cronicle/releases/tag/v0.9.10) -- thank you!

**Comment by dheise-rheinenergie-trading-com** [comment]
Thank you very much, Joseph!
